js_modules/dagit/packages/core/src/ticks/EvaluateSchedule.tsx

```diff
@@ -0,0 +1,400 @@
+            scheduleName: name,
+          },
+          timestamp,
+        },
+      };
+    }, [name, repoAddress, timestamp]),
+  );
+  const [result, setResult] = React.useState<Awaited<
+    ReturnType<typeof scheduleDryRunMutation>
+  > | null>(null);
+  React.useEffect(() => {
+    scheduleDryRunMutation().then((result) => {
+      setResult(() => result);
+    });
+  }, [scheduleDryRunMutation]);
+
+  if (!result || !result.data) {
+    return (
+      <Box padding={32} flex={{justifyContent: 'center', alignItems: 'center'}}>
+        <Spinner purpose="page" />
+      </Box>
+    );
+  }
+
+  const evaluationResult =
+    result?.data?.scheduleDryRun.__typename === 'DryRunInstigationTick'
+      ? result?.data?.scheduleDryRun.evaluationResult
+      : null;
+
+  const innerContent = () => {
+    const data = result.data;
+    if (!data || !evaluationResult) {
+      return (
+        <NonIdealState
+          title="An unknown error occurred"
+          description={
+            <span>
+              File an issue on{' '}
+              <a href="https://github.com/dagster-io/dagster" target="_blank" rel="noreferrer">
+                Github
+              </a>{' '}
+              if you think this is a bug
+            </span>
+          }
+          icon="error"
+        />
+      );
+    } else if (evaluationResult.error) {
+      return <PythonErrorInfo error={evaluationResult.error} />;
+    }
+    if (!evaluationResult.runRequests?.length) {
+      return (
+        <SkipWrapper>{evaluationResult?.skipReason || 'No skip reason was output'}</SkipWrapper>
+      );
+    } else {
+      return (
+        <RunRequestTable
+          runRequests={evaluationResult.runRequests}
+          repoAddress={repoAddress}
+          isJob={true}
+          jobName={jobName}
+          name={name}
+        />
+      );
+    }
+  };
+
+  const numRunRequests = evaluationResult?.runRequests?.length;
+  const error = evaluationResult?.error;
+
+  return (
+    <Box flex={{direction: 'column', gap: 8}}>
+      <Box>
+        <Grid>
+          <div>
+            <Subheading>Result</Subheading>
+            <Box flex={{grow: 1, alignItems: 'center'}}>
+              <div>
+                {error ? (
+                  <Tag intent="danger">Failed</Tag>
+                ) : numRunRequests ? (
+                  <Tag intent="success">{numRunRequests} Run requests</Tag>
+                ) : (
+                  <Tag intent="warning">Skipped</Tag>
+                )}
+              </div>
+            </Box>
+          </div>
+          <div>
+            <Subheading>Tick</Subheading>
+            <Box flex={{grow: 1, alignItems: 'center'}}>
+              <Mono>
+                {timestampToString({
+                  timestamp: {unix: timestamp},
+                  locale,
+                  timezone: userTimezone,
+                  timeFormat: {
+                    showTimezone: true,
+                  },
+                })}
```

Comment:
Could this be `<TimestampDisplay timestamp={timestamp} />`?  It seems like all the pieces of that component are being re-imported and implemented in this file and I _think_ they're the same?

Reply:
I looked at TimestampDisplay and it has those extra icons to the left and right that we don't need. I could update it to make it possible to turn those off. It also has some other unnecessary stuff too I think though